torch/csrc/generic/Storage.h

```diff
@@ -2,10 +2,13 @@
 #define TH_GENERIC_FILE "generic/Storage.h"
 #else
 
+#ifndef TH_GENERIC_STORAGE_DEF
+#define TH_GENERIC_STORAGE_DEF
```

Comment:
Strongly encourage you to just move this out of this header entirely. Then, you can have `torch/csrc/Storage.h` include whatever header you decide to put it in, so you don't have to fix include sites. (And it will only be included once, via usual `pragma once` or include guard shenanigans)